tests/compile-fail/super_trait_not_implemented.stderr

```diff
@@ -1,12 +1,10 @@
 error[E0277]: the trait bound `std::boxed::Box<Dog>: Supi` is not satisfied
-  --> $DIR/super_trait_not_implemented.rs:18:5
+  --> $DIR/super_trait_not_implemented.rs:18:18
    |
+14 | fn requires_foo<T: Foo>(_: T) {}
+   |    ------------    --- required by this bound in `requires_foo`
+...
 18 |     requires_foo(Box::new(Dog)); // shouldn't, because `Box<Dog>: Supi` is not satisfied
-   |     ^^^^^^^^^^^^ the trait `Supi` is not implemented for `std::boxed::Box<Dog>`
+   |                  ^^^^^^^^^^^^^ the trait `Supi` is not implemented for `std::boxed::Box<Dog>`
    |
    = note: required because of the requirements on the impl of `Foo` for `std::boxed::Box<Dog>`
-note: required by `requires_foo`
-  --> $DIR/super_trait_not_implemented.rs:14:1
-   |
-14 | fn requires_foo<T: Foo>(_: T) {}
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
So this is just because of a newer rustc version, right? This change is not caused by any of your changes, am I right?

Reply:
Nothing to do with my changes. Just look at the few lines above it and you'll see that this `note`  is integrated into the error itself:

> 14 | fn requires_foo<T: Foo>(_: T) {}
>      |    ------------    --- required by this bound in `requires_foo`
>    ...